docs/cataloging-sample-data.md

```diff
@@ -6,23 +6,36 @@ The steps below walks you through the creation of STAC Collection and STAC Item.
 
 ## Steps to Catalog Sample data
 
-1. First step in the process is the creation of the [STAC collection](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md). A STAC Collection is created by from a [json document](https://aoigeospatial.blob.core.windows.net/stac/collection_naip_test.json) that is uploaded to the appropriate container in the Storage Account.
+1. First step in the process is the creation of the [STAC collection](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md). A STAC Collection is created from a [json document](../deploy/sample-data/collection_naip_test.json) that is uploaded to the appropriate container in the Storage Account.
 
 
     ```bash
-    azcopy copy https://aoigeospatial.blob.core.windows.net/stac/collection_naip_test.json ~/data/collection_naip_test.json
+    wget https://raw.githubusercontent.com/Azure/Azure-Orbital-STAC/main/deploy/sample-data/collection_naip_test.json -P ~/data/collection_naip_test.json
     ```
 
     ```bash
     azcopy copy ~/data/collection_naip_test.json "https://<storage account>.blob.core.windows.net/staccollection/collection_naip_test.json?<SAS-token>" --recursive=true
     ```
 
-2. Once the STAC collection is created, the next steps is to create the [STAC Items](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md). Addition of the STAC Item to a STAC Collection is two step process.
+2. Generate the SAS token for getting the data from Planetary Computers using the link https://planetarycomputer.microsoft.com/api/sas/v1/token/naip. Clicking on the link will give an output like this:
+
+```
+{
+    "msft:expiry":"2021-04-08T18:49:29Z",
```

Comment:
we should obfuscate this token though its invalid....

Reply:
done.